src/auth/screens/language-settings.js

```diff
@@ -15,4 +15,8 @@ export default [
     code: 'tr',
     name: 'Türkçe',
   },
+  {
+    code: 'br',
```

Comment:
Because this is not pure Brazilian, the code should be such `pt-br`, this also applies to the file name.

Reply:
I was going to use `pt-br`, but the [instructions](https://github.com/gitpoint/git-point/blob/master/CONTRIBUTING.md#translations) says that we should use a two-letter code from [here](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Officially_assigned_code_elements), so I used only `br`. Should I change?

Reply:
Yea, those instructions should probably be changed. In this case, `pt-br` makes perfect sense.

Reply:
Okay, I will change the code and the file name as soon as I can. Thank you both for the suggestions and corrections.

Reply:
Changes made.